CMakeLists.txt

```diff
@@ -168,12 +167,43 @@ endif()
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src)
 
-if(FIRST_CUDA)
+if(USE_CUDA)
+  find_package(CUDA REQUIRED)
+  add_definitions(-DMSHADOW_USE_CUDA=1)
+  if(FIRST_CUDA AND (NOT USE_OLDCMAKECUDA))
+    set(__cuda_toolset "7.5" "8.0" "9.0")
```

Comment:
latest is 9.1. does the support for new cuda version have to be manually added?

Reply:
I don't actually know -- this was not written by me. It was moved from somewhere else in this file.
I don;t actually know what this section does. I suppose that I can add 9.1, however.

Reply:
@DickJC123 would you recommend how this section should be written?